Microsoft.Toolkit.Uwp.UI.Controls/HeaderedTextBlock/HeaderedTextBlock.cs

```diff
@@ -39,6 +41,14 @@ private void UpdateHeader()
             }
         }
 
+        private void UpdateText()
+        {
+            if (_textContent != null)
```

Comment:
Yep, the intent of the commit was to give the Text TextBlock the same behavior of the Header TextBlock.